consensus/spos/bls/sovereignSubRoundEnd.go

```diff
@@ -110,9 +105,21 @@ func (sr *sovereignSubRoundEnd) getAllOutGoingOperations(currentOperations *sove
 		outGoingOperations = append(outGoingOperations, unconfirmedOperations...)
 	}
 
+	log.Debug("current outgoing operations", "hash", currentOperations.Hash)
 	return append(outGoingOperations, currentOperations)
 }
 
+func (sr *sovereignSubRoundEnd) sendOutGoingOperations(ctx context.Context, data []*sovereign.BridgeOutGoingData) {
+	resp, err := sr.bridgeOpHandler.Send(ctx, &sovereign.BridgeOperations{
+		Data: data,
+	})
+	if err != nil {
+		log.Error("sovereignSubRoundEnd.doSovereignEndRoundJob.bridgeOpHandler.Send", "error", err)
```

Comment:
missing return

Reply:
Yes, good catch.
This was however fixed on this branch already: https://github.com/multiversx/mx-chain-go/pull/5763/files#diff-e6295dd6812e5903f1738d649df21df90a0371ffd1a0106fe17c6b1a56c1332b